packages/log/0.123.4/dataset/log/agent/stream/stream.yml.hbs

```diff
@@ -0,0 +1,9 @@
+type: log
```

Comment:
Interesting version 🤔

Reply:
Yes this is totally fake, do not merge it please :D